designer/server/src/main/scala/pl/touk/nussknacker/ui/api/description/StatisticsApiEndpoints.scala

```diff
@@ -0,0 +1,49 @@
+package pl.touk.nussknacker.ui.api.description
+
+import derevo.circe.{decoder, encoder}
+import derevo.derive
+import pl.touk.nussknacker.restmodel.BaseEndpointDefinitions
+import sttp.model.StatusCode.Ok
+import sttp.tapir._
+import sttp.tapir.EndpointIO.Example
+import sttp.tapir.derevo.schema
+import sttp.tapir.json.circe.jsonBody
+import sttp.tapir.{PublicEndpoint, statusCode}
+
+class StatisticsApiEndpoints extends BaseEndpointDefinitions {
+
+  import StatisticsApiEndpoints.Dtos._
+
+  lazy val statisticUrlEndpoint: PublicEndpoint[Unit, Unit, StatisticUrlResponseDto, Any] =
+    baseNuApiEndpoint
+      .summary("Statistics URL service")
+      .tag("Statistics")
+      .get
+      .in("statistic" / "url")
```

Comment:
WDYT about abstracting over the current implementation with the URL? I'm talking about the name of the endpoint. Like eg. "statistics/usage"?

Reply:
"statistics/usage" sounds great